### DISCUSSION TITLE
OGS post

### DISCUSSION THREAD

**Discussion by changchengdouletmedia** [changcheng967/Kata_speed#113] [General]
Put the OGS announce here

**Comment by changchengdouletmedia** [comment]
blacklist: ["Tilano",
            "ujykfyijhgf",
            
            ],

**Reply by changchengdouletmedia** [reply]
that is how the ogs blacklist supposed to be like

**Reply by changcheng967** [reply]
gotcha

**Comment by changcheng967** [comment]
# Post announce here!

**Comment by changcheng967** [comment]
https://online-go.com/welcome/?linked-challenge=qBJxy8iPDuPMueCrmHijnK

**Reply by changcheng22023** [reply]
Good

**Reply by changcheng22023** [reply]
# Join here!
## @changcheng22023 @changcheng6541 @DouletMedia

**Reply by changcheng22023** [reply]
Join here!

**Comment by changcheng22023** [comment]
Is this like show and tell?

**Reply by changcheng967** [reply]
kind of

**Comment by changcheng967** [comment]
### https://online-go.com/game/66972681

**Comment by changcheng967** [comment]
# [Leela Zero Strong](https://online-go.com/player/1591164/) is on fire!!!
## He won with 50 opponent lead! [Game link here](https://online-go.com/game/66972681)

**Reply by changchengdouletmedia** [reply]
wow!

**Reply by changcheng22023** [reply]
So freaking good

**Reply by changcheng22023** [reply]
> # [Leela Zero Strong](https://online-go.com/player/1591164/) is on fire!!!
> ## He won with 50 opponent lead! [Game link here](https://online-go.com/game/66972681)

SO good bro!

**Comment by changchengdouletmedia** [comment]
Successfully beaten special trained 9 network https://online-go.com/game/66972959

**Comment by changchengdouletmedia** [comment]
https://github.com/changchengdouletmedia/OGS-SGF-share/blob/main/66973076-102-00bStrongBot-Golaxy%209D.sgf

**Reply by changchengdouletmedia** [reply]
Golaxy 9D vs 00bstrongbot

**Comment by changcheng967** [comment]
https://github.com/changchengdouletmedia/OGS-SGF-share/blob/main/66973273-085-00bStrongBot-Golaxy%209D.sgf

**Comment by changcheng967** [comment]
newest

**Comment by changchengdouletmedia** [comment]
https://github.com/changchengdouletmedia/OGS-Doulet-Media/pull/9
